sbws/lib/v3bwfile.py

```diff
@@ -135,12 +145,21 @@
     'relay_recent_measurement_failure_count',
     # The number of success results should be:
     # the number of attempts - the number of failures
+    # Number of error results, it can be deduced counting the number of errors
+    # but this way seem clearer.
+    'relay_recent_measurement_exclusion_not_success_count',
```

Comment:
```suggestion
    'relay_recent_measurement_exclusion_count',
```

Reply:
I left commented that key because that would be the sum of all the excluded measurements.
For instance, if each rule would exclude a measurement.
What about `relay_recent_measurement_exclusion_error_count`?

Reply:
This is actually the sum of all the errors, which we can count externally, so maybe just not include it?

Reply:
Ok

Reply:
or maybe leaving it, it makes it more clear that failure (when the relay was attempted to be measured but we don't know what happened) is different from error